radio/src/storage/conversions/datastructs_219.h

```diff
@@ -28,7 +28,7 @@
   #define NUM_SWITCHES_219                 10
 #elif defined(PCBXLITE) ||defined(PCBXLITES)
   #define NUM_SWITCHES_219                  6
-#elif  defined(RADIO_TLITE) || defined(RADIO_FAMILY_JUMPER_T12) || defined(RADIO_T8) || defined(PCBNV14)
+#elif  defined(RADIO_TLITE) || defined(RADIO_FAMILY_JUMPER_T12) || defined(RADIO_T8) || defined(PCBNV14) || defined(RADIO_COMMANDO8)
```

Comment:
Please remove all changes in `storage/conversions/*`: you would only need them to support a pre-existing binary storage. As EdgeTx does not support binary storage for new radios, it should not be needed.